GpxKit/Gpx+Parser.swift

```diff
@@ -24,12 +32,22 @@ public extension Gpx {
 
 fileprivate extension XMLIndexer {
 
+    func metadata() -> Metadata? {
+        guard element != nil else { return nil }
```

Comment:
Does Xcode give a warning here that element is unused? Why do you need the check?

Reply:
There may not be a `<metadata>` element in the XML in which case `element` here is `nil`. Indexing into a document with SWXMLHash gives you a reference to an element that may or may not exist. This simply returns early. You're right, we don't need it. Shall I remove it?

Reply:
Um nah that seems fine 👍